RealEstate/apps/core/templates/core/realtorHome.html

```diff
@@ -1,5 +1,56 @@
 {% extends "core/base.html" %}
 
-{% block body %}
-  <p>Home page for realtors.</p>
+{% block style %}
+<style>
+.vcenter {
+	display: inline-block;
+	vertical-align: middle;
+	float: none;
+	margin-left:-3px;
+}
+</style>
 {% endblock %}
+
+{% block body %}
+    <h2><strong> Dashboard </strong> </h2>
+    <ul class="list-group">
```

Comment:
Let's also provide some space between the header and list-group. Something around 60px. To do that inline for the header: style="margin-bottom : 60px;"